fetch/programmer-humour/script.js

```diff
@@ -0,0 +1,27 @@
+
+let myData;
+
+async function fetchData() {
+    try{
+        const response = await fetch("https://xkcd.now.sh/?comic=latest")
```

Comment:
Please, add closing semicolon at the end of your expressions. Though your JS will work, it prevents you from unexpected errors.